test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -509,19 +511,48 @@ void RandomCodeOptions::setWeight(eth::Instruction _opCode, int _weight)
 	setWeights();
 }
 
-void RandomCodeOptions::addAddress(Address const& _address)
+void RandomCodeOptions::addAddress(Address const& _address, AddressType _type)
 {
-	addressList.push_back(_address);
+	switch(_type)
+	{
+		case AddressType::ACCOUNT:
+			accountAddressList.push_back(_address);
+			callAddressList.push_back(_address);
+		break;
```

Comment:
The indentation of `break;` look weird to me.  It is just another statement following the previous one.